command/meta.go

```diff
@@ -76,3 +76,44 @@ func (m *meta) client() *api.Client {
 		Port: *m.port,
 	}, nil)
 }
+
+// requestUserApproval returns an exit code and boolean describing if the user
+// approved. If the user did not approve (false is returned), exit code is provided.
+func (m *meta) requestUserApproval(taskName string) (int, bool) {
+	m.UI.Info("Enabling the task will perform the actions described above.")
+	m.UI.Output(fmt.Sprintf("Do you want to perform these actions for '%s'?", taskName))
+	m.UI.Output(" - This action cannot be undone.")
+	m.UI.Output(" - CTS cannot guarantee Terraform will perform these exact actions if")
```

Comment:
CTS -> Consul Terraform Sync

CTS is fine for internal comments and colloquial usage, but since this is user-facing I think it would be best to expand to the full name (a few more cases below too).

Reply:
Good point! I will change this to be the full name

Reply:
There are two more occurrences @lornasong then 👍

Reply:
Thanks for pointing that out! Ok! I've pushed a commit to catch the other two occurrences.